docs/README.md

```diff
@@ -0,0 +1,7 @@
+# Swift for VSCode Documentation
+
```

Comment:
Don't think we need this file. It's just another click before someone gets to the documentation they want. The individual docs can go in the repository readme.